docs/guides/preinstall.md

```diff
@@ -4,77 +4,111 @@ These steps are common to most distros. Please refer to both this page and this
+8. **Size:** pick the **desired amount of space** for Linux, because you will **not be able** to change it.
+9. If you want separate partitions for `/home`, `/boot` etc., create them as well (if you are a beginner and you don't understand this point, you may just skip it).
+
+!!! Note "Triple booting with Linux and Windows"
     Refer to [this guide](https://wiki.t2linux.org/guides/windows/) to make sure you will be able to boot both Windows and Linux.
 
 ## Create Linux installation USB
 
 ### Selecting an ISO
 
-If there is an ISO with T2 support for your distro, you can download it here:
+Listed below are the currently available installer ISOs for download:
 
-| Linux Distribution | Install ISO with T2 support |
-| ------------------ | --------------------------- |
+| Linux Distribution | Download ISO with T2 support |
+| ------------------ | ---------------------------- |
 | Arch Linux         | <https://github.com/t2linux/archiso-t2/releases/latest> |
 | blendOS            | <https://docs.blendos.co/guides/installation-guide-for-macs-with-t2-chip> |
 | EndeavourOS        | <https://github.com/t2linux/EndeavourOS-ISO-t2/releases/latest> |
-| Fedora       | <https://github.com/mikeeq/mbp-fedora> |
-| Fedora       | <https://github.com/t2linux/fedora-iso/releases/latest> |
+| Fedora             | <https://github.com/mikeeq/mbp-fedora> |
+| Fedora             | <https://github.com/t2linux/fedora-iso/releases/latest> |
 | Gentoo             | Please refer to this [page](https://wiki.t2linux.org/distributions/gentoo/installation/) |
 | Manjaro            | <https://github.com/NoaHimesaka1873/manjaroiso-t2/releases/latest> |
 | NixOS              | <https://github.com/t2linux/nixos-t2-iso> |
-| Ubuntu             | <https://github.com/t2linux/T2-Ubuntu/releases/latest> |
+| Ubuntu & Kubuntu   | <https://github.com/t2linux/T2-Ubuntu/releases/latest> |
+
+#### Other distributions
+
+If you are a beginner we **highly recommend** to choose one of the distros above.
+
+In case you wish to proceed manually, you can download the official ISO from the distro website, but you will have to use an external USB keyboard and mouse for the install process. Additionally, you may require a wired internet connection with a USB adapter.
+
+Please, refer to this guide: [Basic setup](https://wiki.t2linux.org/guides/postinstall/).
 
-For other distros, you can download the distro's normal install ISO, but you will have to use an external USB keyboard and mouse for this install process, and you may need a wired internet connection. Additionally, if you later have issues with installing the bootloader (Eg :- GRUB), try booting the ISO with the `efi=noruntime` kernel parameter.
+You can also find detailed info for unsupported debian-based distros [here](https://wiki.t2linux.org/distributions/debian/installation/).
```

Comment:
Consider the fact that the wiki is not only about debian based distros, so the previous wording looks good to me.

Reply:
That's why there is the link to the Basic setup guide (postinstall.md)
I think the linking of the two guides can replace those lines:
```
Please, refer to this guide: [Basic setup](https://wiki.t2linux.org/guides/postinstall/).
You can also find detailed info for unsupported debian-based distros [here](https://wiki.t2linux.org/distributions/debian/installation/).
```
Basic setup explains the procedure for non-T2 iso
More specifically there is a guide for debian-based non supported distro, where it is also mentioned about the efi=noruntime parameter.

What do you think?

Reply:
What I meant was that remove the word *debian-based*. Whatever you suggested for not supported distris is applicable for non-debian based unsupported distros as well.

Reply:
I just took it from the introduction: _This guide shall help you in installing a Debian or Ubuntu based Linux distro, which is not available as a modified ISO with T2 support._

What would like to modify?

Reply:
> What I meant was that remove the word _debian-based_. Whatever you suggested for not supported distris is applicable for non-debian based unsupported distros as well.

Should be good now: [0cf920b](https://github.com/gianmaticus/wiki/commit/0cf920b809b0dacc44977be2f11e2ec09755abae)

Reply:
Are we modifying the debian guide or preinstall guide. You've yourself added the link of the debian guide here, I believe the previous original one was better.

Reply:
Oh, I overlooked the heading, this seems good now.

Reply:
> Are we modifying the debian guide or preinstall guide. You've yourself added the link of the debian guide here, I believe the previous original one was better.
Reverted [e6e0838](https://github.com/gianmaticus/wiki/commit/e6e0838e5e772577acfea012b6bb449eec8301fd)